docs/metrics/customizing-the-sdk/README.md

```diff
@@ -403,6 +403,61 @@ AnotherFruitCounter.Add(4, new("name", "mango"), new("color", "yellow")); // Not
 streams. There is no ability to apply different limits for each instrument at
 this moment.
 
+### Exemplars
+
+Exemplars are example data points for aggregated data. They provide access to
+the raw measurement value, time stamp when measurement was made, and trace
```

Comment:
curious - do we use timestamp or time stamp? (I don't know)

Reply:
timestamp. time stamp was my typo!

Reply:
However...
![image](https://user-images.githubusercontent.com/5232798/221069969-2e8181b3-d38d-4fe2-9204-be167d93968e.png)